mmeds/database.py

```diff
@@ -704,6 +710,14 @@ def build_table_sql(self, table):
         foreign_keys = list(filter(lambda x: '_has_' not in x,
                                    list(filter(lambda x: '_id' in x,
                                                all_cols))))
+        # For the SubjectType table one of the keys will be NULL depending on
+        # if the metadata is for an Animal subject or a human subject
+        if table == 'SubjectType':
+            if self.df['SubjectType']['SubjectType'].iloc[self.row] == 'Human':
+                del foreign_keys[foreign_keys.index('AnimalSubjects_idAnimalSubjects')]
+            else:
```

Comment:
I would consider changing the else to an elif:
```elif self.df['SubjectType']['SubjectType'].iloc[self.row] == 'Animal':```
It's less concise, but more plain to someone reading the code.

Reply:
Yeah that's what I did elsewhere. Must've missed it here. I don't necessarily think it's 'clearer' but more likely to catch a problem in an informative location that pass it along silently. Also if you use suggestions, I can add the code you write directly. Isn't necessary here but they've finally implemented them for multiple lines.

Reply:
Will do, I just actually looked up how to create suggested changes.